modules/sharedaddy/sharedaddy.php

```diff
@@ -68,7 +68,7 @@ function sharing_email_send_post_content( $data ) {
 	$content  = sprintf( __( '%1$s (%2$s) thinks you may be interested in the following post:', 'jetpack' ), $data['name'], $data['source'] );
 	$content .= "\n\n";
 	$content .= $data['post']->post_title."\n";
-	$content .= get_permalink( $data['post']->ID )."\n";
+	$content .= apply_filters( 'sharing_permalink', get_permalink( $data['post']->ID ) ) . "\n";
```

Comment:
This looks good-- I would change the filter name to `jetpack_sharing_permalink`, though.  Namespacing FTW!